test/javascript/components/modals/FinishMentorDiscussionModal.test.js

```diff
@@ -119,9 +119,11 @@ test('shows generic error', async () => {
     />
   )
   await expectConsoleError(async () => {
-    userEvent.click(
-      await screen.findByRole('button', { name: 'End discussion F3' })
-    )
+    await act(async () => {
+      userEvent.click(
+        await screen.findByRole('button', { name: 'End discussion F3' })
+      )
+    })
```

Comment:
This test starting failing for me locally. It's also the one @ErikSchierboom has had problems with. I think this fixes it now. Erik - could you confirm pls?